org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -696,11 +706,9 @@ private void validateAndRun(Set<LFTest> tests, Configurator configurator, TestLe
                     test.result = Result.TEST_PASS;
                 }
 
-            } catch (AssertionError e) {
-                // Do not report assertion errors. They are pretty printed
-                // during reporting.
-            } catch (Exception e) {
-                test.issues.append(e.getMessage());
+            } catch (TestExecutionException e) {
```

Comment:
Nit: the `execute` phase of the test is the only one that will not throw a TestExecutionException, which seems to contradict the naming.

Reply:
Do you have a name proposal? I chose the name to indicate an error during the execution of one of our tests, where executing a test is comprised of validating the code, generating target code, compiling and running the binary.